SofaKernel/modules/SofaHelper/src/sofa/helper/ComponentChange.cpp

```diff
@@ -109,6 +109,10 @@ std::map<std::string, ComponentChange> uncreatableComponents = {
     {"VisualManagerPass", Pluginized("v19.06", "SofaOpenglVisual")},
     {"VisualmanagerSecondaryPass", Pluginized("v19.06", "SofaOpenglVisual")},
 
+    // SofaDenseSolver was pluginized in #1299
+    {"LULinearSolver", Pluginized("v20.06", "SofaDenseSolver")},
+    //{"NewMatCholeskySolver", Pluginized("v20.06", "SofaDenseSolver")},
+    //{"NewMatCGLinearSolver", Pluginized("v20.06", "SofaDenseSolver")},
```

Comment:
Why commented?

Reply:
Yes, that's the point of adding these two lines. Still don't understand why they are commented out though.

Reply:
No aliases (I think you cannot aliases a SofaObject on a particular template by the way)
The problem of ComponentChange is that you cannot deprecate a SofaObject on a particular template, e.g it is no possible to deprecate only the NewMat template of CGLinearSolver.
So I let these two lines commented (as NewMatCholeskySolver and NewMatCGLinearSolver do not exist) to keep the deprecations in mind or if ComponentChange handles template in the future.
Of course, I can just remove these two lines anyway if it is bothering.